.werft/installer-tests.ts

```diff
@@ -468,40 +474,41 @@ function randOsVersion(): string {
     return randomize(options);
 }
 
-function cleanup() {
+async function cleanup(): Promise<void> {
     const phase = INFRA_PHASES["DESTROY"];
     werft.phase(phase.phase, phase.description);
 
-    const ret = callMakeTargets(phase.phase, phase.description, phase.makeTarget);
+    return await callMakeTargets(phase.phase, phase.description, phase.makeTarget)
```

Comment:
You don't need the `await` here as `callMakeTargets` is already a promise - it doesn't hurt either though so feel free to keep it